content/en/monitors/service_level_objectives/_index.md

```diff
@@ -165,33 +165,36 @@ You can also query the Event Explorer programmatically using the [Datadog Events
 
-To proactively manage the configurations of your SLOs, set an [Event Monitor][14] to notify you when events corresponding to certain tags occur.
-
 For example, if you wish to be notified when a specific SLO's configuration is modified, set an Event Monitor to track the text `[SLO Modified]` over the tags `audit,slo_id:<SLO ID>`.
 
 {{< img src="monitors/service_level_objectives/slo-event-monitor.png" alt="SLO event monitor"  >}}
 
 ## SLO widgets
 
+To proactively manage the configurations of your SLOs, set an [Event Monitor][14] to notify you when events corresponding to certain tags occur.
+
 After creating your SLO, you can use the SLO Summary dashboard widget to visualize the status of an SLO along with your dashboard metrics, logs and APM data. For more information about SLO Widgets, see the [SLO Widgets documentation][2] page.
 
 ## SLO status corrections
 
-Status corrections allow you to specify time periods such as planned maintenance that an SLO excludes from its status and error budget calculations.
+Status corrections allow you to exclude specific time periods from SLO status and error budget calculations. This way, you can:
+- Prevent expected downtime, such as scheduled maintenance, from depleting your error budget
+- Ignore non-business hours, where you're not expected to conform to your SLOs
+- Ensure that temporary issues caused by deployments do not negatively impact your SLOs
 
-When you create a correction window for an SLO, the time period you specify is removed from the SLO’s calculation.
-- For monitor-based SLOs, time in the correction window is not counted.
+When you apply a correction, the time period you specify is dropped from the SLO’s calculation.
+- For monitor-based SLOs, the correction time window is not counted.
 - For metric-based SLOs, all good and bad events in the correction window are not counted.
 
-You have the option to create one-time corrections for ad-hoc adjustments, or recurring corrections for predictable adjustments that occur on a regular cadence. One-time corrections require a start and end time, while recurring corrections require a start time, duration, and interval. Recurring corrections are based on [iCalendar RFC 5545's RRULE specification][15]. Specifying an end date for recurring corrections is optional in case you need the correction to repeat indefinitely.
+You have the option to create one-time corrections for ad-hoc adjustments, or recurring corrections for predictable adjustments that occur on a regular cadence. One-time corrections require a start and end time, while recurring corrections require a start time, duration, and interval. Recurring corrections are based on [iCalendar RFC 5545's RRULE specification][15]. The rules that are currently supported are `FREQ`, `INTERVAL`, `COUNT`, and `UNTIL`. Specifying an end date for recurring corrections is optional in case you need the correction to repeat indefinitely. 
```

Comment:
```suggestion
You have the option to create one-time corrections for ad-hoc adjustments, or recurring corrections for predictable adjustments that occur on a regular cadence. One-time corrections require a start and end time, while recurring corrections require a start time, duration, and interval. Recurring corrections are based on [iCalendar RFC 5545's RRULE specification][15]. The supported rules are `FREQ`, `INTERVAL`, `COUNT`, and `UNTIL`. Specifying an end date for recurring corrections is optional in case you need the correction to repeat indefinitely. 
```